bigbluebutton-html5/imports/ui/components/breakout-room/component.jsx

```diff
@@ -200,7 +200,7 @@ class BreakoutRoom extends Component {
           {intl.formatMessage(intlMessages.breakoutRoom, breakout.sequence.toString())}
           <span className={styles.usersAssignedNumberLabel}>
             (
-            {breakout.users.length}
+            {new Set(breakout.users.map(user => user.intId)).size}
```

Comment:
I see what you are doing but what is the reasoning behind it?
Was passing the length as a prop a stable alternative?

Reply:
The problem is related to users who receive multiple invitations. So I just need to get an ID for each user. I did that way because I get this information just when `map` the breakout rooms list.